docs/guides/nextjs.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Setup with Next.js
+nav: 21
+---
+
+[Next.js](https://nextjs.org) is a popular server side rendering framework for React that presents
```

Comment:
Small nit. "Server-side" is a compound adjective meaning there needs to be a hyphen. Next.js follows the grammar rule. See: https://nextjs.org/docs/pages/building-your-application/rendering/server-side-rendering
```suggestion
[Next.js](https://nextjs.org) is a popular server-side rendering framework for React that presents
```